prowler/providers/aws/services/iam/iam_avoid_root_usage/iam_avoid_root_usage.py

```diff
@@ -27,23 +27,23 @@ def execute(self) -> Check_Report_AWS:
                             datetime.datetime.now()
                             - datetime.datetime.strptime(
                                 user["password_last_used"],
-                                "%Y-%m-%dT%H:%M:%S+00:00",
+                                "%Y-%m-%dT%H:%M:%SZ",
```

Comment:
Python's https://dateutil.readthedocs.io/en/stable/ trats both formats equally, could be good to change it to prevent this from happening again.
`parser.parse(timestamp)`

Reply:
Done!